addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -2123,218 +2124,214 @@ we-select.o_we_border_preview_aligned_select {
-                    @include o-position-absolute($o-we-handles-offset-to-hide - $o-we-handle-border-width, auto, auto, $o-we-handle-border-width + 1);
-                    height: $o-we-handle-edge-size;
+                    inset: $o-we-handles-offset-to-hide auto auto 0;
+                    transform: translate(-50%, -50%);
                     cursor: nwse-resize;
-
-                    &:after {
-                        @include o-position-absolute($left: 50%, $bottom: 40%);
-                    }
                 }
 
-                &.ne,
-                &.sw {
-                    content: "\f065";
-                }
-
-                &.nw,
-                &.se {
-                    content: "\f065";
-                    transform: rotate(90deg);
-                }
-
-                &::after {
-                    z-index: 1;
+                .o_handle_indicator {
+                    position: absolute;
+                    inset: $o-we-handles-btn-size * -0.5;
                     display: block;
-                    width: $o-we-handles-btn-size;
                     height: $o-we-handles-btn-size;
-                    border: solid 1px darken($o-we-handles-accent-color, 20%);
-                    line-height: $o-we-handles-btn-size - 2;
-                    font-size: 14px;
-                    font-family: FontAwesome;
-                    background-color: darken($o-we-handles-accent-color, 10%);
-                    color: white;
-                }
-
-                &.o_handle_start:after {
-                    background-color: rgba($o-we-sidebar-content-field-clickable-bg, .6);
-                }
-
-                &:hover,
-                &.o_active {
-                    background-color: rgba($o-we-handles-accent-color, 0.2);
+                    width: $o-we-handles-btn-size;
+                    margin: auto;
+                    border: solid $o-we-handle-border-width $o-we-handles-accent-color;
+                    border-radius: $o-we-handles-btn-size;
+                    background: $o-we-fg-lighter;
+                    outline: $o-we-handle-inside-line-width solid $o-we-fg-lighter;
+                    outline-offset: -($o-we-handles-btn-size * 0.5);
+                    transition: $transition-base;
 
-                    &::after {
-                        border-color: darken($o-we-handles-accent-color, 10%);
-                        background-color: darken($o-we-handles-accent-color, 20%);
+                    &:before {
+                        position: absolute;
+                        display: block;
+                        border-radius: $o-we-handles-btn-size;
+                        inset: -$o-we-handles-btn-size;
+                        content: '';
                     }
                 }
-
-                &.w:after,
-                &.e:after {
-                    content: "\f07e";
+                
+                body:not([class$="-important"]) &:hover, &:active {
```

Comment:
I've updated the javascript part ✅ .

>About this specific SCSS, is there something missing in here? Not tested

This was intentional, if you already click on an element and then move the mouse, it will only trigger the hover state of other elements but not the active state.